ctt/uploaders.py

```diff
@@ -179,13 +179,51 @@ def process(self, processing_job, target_as_source=False):
             upload_request=upload_request
         )
 
+class TagAsLatestUploader:
+    """
+    Tags the image with the 'latest' tag
+    """
+    tag = 'latest'
+
+    def process(self, processing_job, target_as_source=False):
+        if processing_job.resource.type is not cm.ResourceType.OCI_IMAGE:
+            raise NotImplementedError
+        if not target_as_source:
+            src_ref = processing_job.resource.access.imageReference
+        else:
+            src_ref = processing_job.upload_request.target_ref
+        src_prefix, src_name, _ = oci.client._split_image_reference(src_ref)
+        src_name = ci.util.urljoin(src_prefix, src_name)
+        tgt_tag = self.tag
+        tgt_ref = ':'.join((src_name, tgt_tag))
+        upload_request = dataclasses.replace(
+            processing_job.upload_request,
+            source_ref=processing_job.resource.access.imageReference,
+            target_ref=tgt_ref,
+        )
+        # propagate changed resource
+        processing_job.processed_resource = dataclasses.replace(
+            processing_job.resource,
+            access=cm.OciAccess(
+                imageReference=tgt_ref,
+            ),
+            labels=labels_with_migration_hint(
+                resource=processing_job.resource,
+                src_img_ref=processing_job.resource.access.imageReference,
+            ),
+        )
+        return dataclasses.replace(
+            processing_job,
+            upload_request=upload_request
+        )
 
 class DigestUploader:
     '''
     sets `reference_target_by_digest` attribute in upload-request, which will result in
     target-component-descriptor's resouce's access use digest rather than tag to reference
     oci image.
     '''
+
```

Comment:
please do not add formatting changes into commit with different focus

Reply:
This got auto-formatted.